mrp_production_grouped_by_product/tests/test_mrp_production_grouped_by_product.py

```diff
@@ -15,9 +16,19 @@ def setUpClass(cls):
         super(TestProductionGroupedByProduct, cls).setUpClass()
         cls.ProcurementGroup = cls.env['procurement.group']
         cls.MrpProduction = cls.env['mrp.production']
+        cls.IrSecuence = cls.env['ir.sequence']
```

Comment:
s/IrSecuence/IrSequence

Reply:
I'll review it.

Reply:
I think they did it because they can't do `cls.env.ref('mrp.REF')` Another way would be to do ```
cls.picking_type = cls.env['stock.picking.type'].search(
            [('code', '=', 'mrp_operation')], limit=1
        )
```
 Do I change it for the search?